frontend/app/work_packages/controllers/details-tab-overview-controller.js

```diff
@@ -66,36 +68,13 @@ module.exports = function(
         }
       });
       otherGroup.attributes.sort(function(a, b) {
-        return getLabel(a).toLowerCase().localeCompare(getLabel(b).toLowerCase());
+        var getLabel = function(wp) {
+          return function(field) { return vm.getLabel(wp, field); }
+        }(vm.workPackage);
```

Comment:
Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.